packages/upgrade/transforms/__testfixtures__/deprecate-light-prop.output.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Button, Layer } from '@carbon/react';
+
+function TestComponent() {
+  <div>
+    <Layer><Button>Click me</Button></Layer>
+    <Button>Another button</Button>
+    <Layer>
+      <Button>Nested light button</Button>
+    </Layer>
```

Comment:
The input for this piece in the other file is

```
    <Layer>
      <Button light>Nested light button</Button>
    </Layer>
```

If they're still using `light` inside a `Layer` would we want to nest it inside another layer instead?

```suggestion
    <Layer>
      <Layer>
        <Button>Nested light button</Button>
      </Layer>
    </Layer>
```

but I might be wrong here, not sure if you already tested this. Let me know what you find.